#command
uint16 Command
uint8 Option1
uint8 Option2
uint8 Option3
string CommandText
string Description
